src/tests/ftest/util/general_utils.py

```diff
@@ -30,12 +30,13 @@
 import string
 from pathlib import Path
 from errno import ENOENT
-from time import sleep
+from time import sleep, time
 
 from avocado import fail_on
 from daos_api import DaosApiError, DaosServer, DaosContainer, DaosPool
 from ClusterShell.Task import task_self
 from ClusterShell.NodeSet import NodeSet
+from command_utils import CommandFailure
```

Comment:
(pylint-no-name-in-module) No name 'CommandFailure' in module 'command_utils'

Reply:
Did you forget to `git add` a file?